lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -78,6 +78,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
     async acquireTokenSilent(request: SilentRequest): Promise<AuthenticationResult> {
+        const endMeasurement = this.performanceManager.startMeasurement("acquireTokenSilent");
```

Comment:
For these can we pass in the same ApiId we're using for the call?  ApiId.acquireTokenSilent_silentFlow in this case?

Reply:
For now, these labels are just used for label the measurements that show up in the browser performance profile. We can easily include more data once we start sending this data to the server.

Reply:
@hectormmg Might be good to show a screenshot of these annotations to provide context.